src/parse_file.cpp

```diff
@@ -7,16 +7,31 @@
 // identity
 #include "orc/parse_file.hpp"
 
+// application config
+#include "orc/features.hpp"
+
 // stdc++
 #include <bit>
 #include <cstdio>
 
 // system
 #include <fcntl.h> // open
+#include <sys/mman.h>
+#include <unistd.h> // close#if ORC_FEATURE(MACH_O)
+
+#if ORC_FEATURE(MACH_O)
 #include <mach-o/fat.h>
 #include <mach-o/loader.h>
-#include <sys/mman.h>
-#include <unistd.h> // close
+#else
+#define	MH_MAGIC 0xfeedface
```

Comment:
I'd prefer `constexpr uint64_t FOO = 0xwhatever` if there isn't a reason for #defines. If there is, can we clean up formatting?
````
#else
#    define FOO foo
etc.
````

Reply:
Making these constexpr is the better path. I'll do that.